src/main/steps/applicant1/review-your-application/content.ts

```diff
@@ -80,7 +81,7 @@ const en = ({ isDivorce, isApplicant2, userCase, partner, required, isJointAppli
   otherCourtCasesLine2: `The applicant has given details of other court cases relating to the ${
     isDivorce ? 'marriage' : 'civil partnership'
   }:`,
-  applicantLegalProceedingsDetails: userCase.applicant1LegalProceedingsDetails,
+  applicantLegalProceedingsDetails: `${userCase.applicant1LegalProceedingsDetails}`,
```

Comment:
how comes you added this?

Reply:
i assume its the same for the welsh

Reply:
Hi @velimd thanks for finding this, yes I removed the template literal.